libmount/src/hook_mount.c

```diff
@@ -415,7 +415,6 @@ static int set_vfsflags(struct libmnt_context *cxt,
 		rc = api->fd_tree = open_mount_tree(cxt, NULL, (unsigned long) -1);
 		if (rc < 0)
 			return rc;
-		rc = 0;
 	}
```

Comment:
Seems incorrect. The function  open_mount_tree() returns non-zero on success, but the following code assumes rc is 0.

Reply:
Indeed, dropped.